libs/sdk-ui-dashboard/src/model/events/userInteraction.ts

```diff
@@ -1,4 +1,5 @@
 // (C) 2021-2023 GoodData Corporation
+import { ShareDialogInteractionData } from "@gooddata/sdk-ui-kit";
```

Comment:
I am not sure about this reference @kandl What you think about it that we have some type in model referenced to UI-Kit ? Not like it ...

Reply:
That's right, we should not couple "dashboard model" with "sdk-ui-kit model"

Reply:
I copied all the types to both location. There are less event, but the payload is bigger now. Would it be better to utilise some `any` type? 🤔